examples/spark-sql/pom.xml

```diff
@@ -34,7 +34,7 @@
         <sedona.scope>compile</sedona.scope>
         <geotools.scope>compile</geotools.scope>
 
-        <geotools.version>28.2</geotools.version>
```

Comment:
Let's not change the example project since it relies on a released version of Sedona